stock_production_lot_multi_company/models/stock_production_lot.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Ainara Galdona - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields
+
+
+class StockProductionLot(models.Model):
+
+    _inherit = 'stock.production.lot'
+
+    company_id = fields.Many2one(comodel_name='res.company', string='Company')
+
+    _defaults = {
+        'company_id': lambda self, cr, uid, c:
```

Comment:
Can you add a comment with link reference to odoo issue to future refactory change it to `default=` param?